scripts/lib/install-node

```diff
@@ -20,20 +20,23 @@ if node_wrapper_path="$(type -p node)"; then
     current_node_version="$(node --version)"
 fi
 
-if [ "$($YARN_BIN --version 2>/dev/null)" = "$yarn_version" ] && [ "$current_node_version" = "v$node_version" ] && [ -L "$node_wrapper_path" ]; then
+if [[ "$($YARN_BIN --version 2>/dev/null)" == "$yarn_version" \
+          && "$current_node_version" == "v$node_version" \
+          && -L "$node_wrapper_path" ]]; then
     echo "Node version $node_version and yarn version $yarn_version are already installed."
     exit 0
 fi
 
-if [ "$current_node_version" != "v$node_version" ] || ! [ -L "$node_wrapper_path" ]; then
+if [[ "$current_node_version" != "v$node_version" || ! -L "$node_wrapper_path" ]]; then
     export NVM_DIR=/usr/local/nvm
-    if ! [ -e "$NVM_DIR/nvm.sh" ]; then
-        wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+    if  [[ ! -e "$NVM_DIR/nvm.sh" ]]; then
+        wget -O- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
     fi
 
     source "$NVM_DIR/nvm.sh"
     nvm install "$node_version" && nvm alias default "$node_version"
-    export NODE_BIN="$(nvm which default)"
+    export NODE_BIN
+    NODE_BIN=$(nvm which default)
```

Comment:
This change seems not helpful?

Reply:
This change actually is helpful, see [SC2155](https://github.com/koalaman/shellcheck/wiki/SC2155). (At some point we should consider integrating ShellCheck into the linter.)

Reply:
Ahh, fair enough.  It's really easy to integrate new tools into the linter; `git grep pyflakes` if you're interested in seeing how it works.

Reply:
Although in this case it looks like it doesn’t need to be exported at all.

Reply:
#10162 adds ShellCheck as a linter.

Reply:
@andersk great